src/ontology/reports/sync-synonym/review-qc-duplicate-exact-synonym-no-abbrev.tsv

```diff
@@ -1512,9 +1512,9 @@ astrocytic tumor	MONDO:0002505	NCIT:C9022	updated		MONDO:0021636
 astrocytic tumor	MONDO:0021636	DOID:3069	unconfirmed		
 astrocytic tumor	MONDO:0021636	NCIT:C6958	unconfirmed		
 astrocytic tumor	MONDO:0021636	Orphanet:94	confirmed		
-astrocytoma	MONDO:0019781	NCIT:C60781	unconfirmed	MONDO:AMBIGUOUS|MONDO:AMBIGUOUS	
-astrocytoma	MONDO:0019781	https://github.com/NCI-Thesaurus/thesaurus-obo-edition/issues/23	confirmed	MONDO:AMBIGUOUS|MONDO:AMBIGUOUS	
-astrocytoma	MONDO:0021636	Orphanet:94	updated	MONDO:AMBIGUOUS|MONDO:AMBIGUOUS	
+astrocytoma	MONDO:0019781	NCIT:C60781	unconfirmed	MONDO:AMBIGUOUS|MONDO:AMBIGUOUS|http://purl.obolibrary.org/obo/mondo#AMBIGUOUS	
```

Comment:
However this looks like a new bug I'll have to fix. Should not be difficult. There shouldn't be a duplication of synonym types here. Interesting to note that one of them is a CURIE and one is an IRI.

Additionally, we don't strictly need synonym types to show in this template. I just added it because I thought it might help add some context when reviewing these.